documentation/projects/planning.md

```diff
@@ -24,23 +24,53 @@ maintainer in the #openverse channel of the
 ## Where documents are kept, how they are named, and how they are reviewed
 
 For each project, planning documents should be kept in the
-`/documentation/projects/proposals/` directory of this repository. Each project
-should have its own subdirectory. Projects predating this document have been
-updated to follow this structure. Individual documents should be date-stamped in
-the `YYYYMMDD` format. An example of this entire structure follows:
+`/documentation/projects/proposals/` directory of this repository.
+
+- Each project should have its own subdirectory. This is name after the project,
+  converted to `snake_case`.
+- Individual documents should be date-stamped in the `YYYYMMDD` format.
+- This subdirectory must contain an `index.md` file in a
+  [standard format](/projects/templates/index.md).
+- This subdirectory must contain a project proposal in a
+  [standard format](/projects/templates/project_proposal.md). This is named
+  `YYYYMMDD-project_proposal.md`.
+- This subdirectory can contain any number of implementation plans, each in a
+  [standard format](/projects/templates/implementation_plan.md). These are named
+  `YYYYMMDD-implementation_plan_<name>.md` where the `<name>` is the plan name,
+  converted to `snake_case`.
+
+Refer to the existing projects for reference.
+
+````{tip}
+You can initialise a new project using the `just` recipe `init-project`. It
+accepts one argument as the project name followed by any number of
```

Comment:
```suggestion
accepts one argument as the project name followed optionally by any number of
```

I think that would highlight the most common workflow, which would be creating the proposal without the IPs yet -- since they're generally blocked on approval of the proposal, right?

Reply:
This makes sense, updated in 2a777ca.